server/test/server.go

```diff
@@ -62,11 +62,8 @@ func RunQueryTests(t *testing.T, ds storage.OpenFGADatastore) {
 
 func RunCommandTests(t *testing.T, ds storage.OpenFGADatastore) {
 	t.Run("TestWriteCommand", func(t *testing.T) { TestWriteCommand(t, ds) })
-
-	t.Run("TestWriteAuthorizationModel",
-		func(t *testing.T) { TestWriteAuthorizationModel(t, ds) },
-	)
-
+	t.Run("TestWriteAuthorizationModel", func(t *testing.T) { TestWriteAuthorizationModel(t, ds) })
+	t.Run("TestWriteAuthorizationModelWithRelationalTypes", func(t *testing.T) { TestWriteAuthorizationModelWithRelationalTypes(t, ds) })
```

Comment:
Can we refactor this name to `TestWriteAuthorizationModelWithTypeInfo` and the same goes for the actual function name getting called in `server/test/write_authzmodel.go`

Reply:
Done. Thanks!